### PR TITLE
Fix bug - skip k8s and qingcloud test when no client config files

### PR DESCRIPTION
**Pull request by chilianyi**

### CODE REVIEW

File: pkg/cmd/runtime/plugins/k8s/k8s_runtime_test.go [hunk 1]
```diff
@@ -69,6 +72,13 @@ spec:
 `
 
 func TestK8sRuntime(t *testing.T) {
+	clientConf := "~/.kube/config"
+	_, err := os.Stat(strings.Replace(clientConf, "~/", os.Getenv("HOME")+"/", 1))
+	if err != nil {
+		fmt.Printf("K8s runtime test skipped because no [%s] %s", clientConf, err)
```
Comment (chai2010) [inline pkg/cmd/runtime/plugins/k8s/k8s_runtime_test.go]: s/fmt.Printf/t.Logf/

File: pkg/cmd/runtime/plugins/qingcloud/qingcloud_runtime_test.go [hunk 2]
```diff
@@ -136,6 +139,13 @@ const test_appConf = `
 `
 
 func TestQingCloudRuntime(t *testing.T) {
+	clientConf := "~/.qingcloud/config.yaml"
+	_, err := os.Stat(strings.Replace(clientConf, "~/", os.Getenv("HOME")+"/", 1))
+	if err != nil {
+		fmt.Printf("QingCloud runtime test skipped because no [%s] %s", clientConf, err)
```
Comment (chai2010) [inline pkg/cmd/runtime/plugins/qingcloud/qingcloud_runtime_test.go]: ditto

File: pkg/cmd/runtime/plugins/k8s/k8s_runtime_test.go (hunk 1 above)
Comment (chilianyi) [reply pkg/cmd/runtime/plugins/k8s/k8s_runtime_test.go]: t.Logf + t.Skip() is the same with t.Skipf()
But I have tried without ~/.kube/config, no output at all.  
If I use fmt.Printf, run go test -v ./..., I can see the output.

Comment (chai2010) [reply pkg/cmd/runtime/plugins/k8s/k8s_runtime_test.go]: `t.Logf` or `t.Skipf` will print the output when run `go test -v`

Comment (chilianyi) [reply pkg/cmd/runtime/plugins/k8s/k8s_runtime_test.go]: I have tried, but nothing output. like this:
PASS
ok  	openpitrix.io/openpitrix/pkg/cmd/runtime/plugins/k8s	0.076s
PASS
ok  	openpitrix.io/openpitrix/pkg/cmd/runtime/plugins/qingcloud	0.033s


And go test -v , config.test failed

**Pull request merged**